api/app/controllers/spree/api/address_books_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Spree
+  module Api
+    class AddressBooksController < Spree::Api::BaseController
+      # Note: the AddressBook is the resource to think about here, not individual addresses
+
+      before_filter :ensure_api_token
```

Comment:
I believe that this behaviour exists by default. (specs still pass with it removed)